docs/Modpacks/Other-Topics/Adding-and-Removing-Recipes.md

```diff
@@ -90,14 +102,25 @@ ServerEvents.recipes(event => {
         - `.outputFluids()`
         - `.chancedFluidOutput()`
 - Energy:
-    - `.inputEU()`: Makes the recipe consume a lump sum of EU to start the recipe. Most often seen in fusion reactor recipes.
-    - `.outputEU()`: Makes the recipe produce a lump sum of EU upon recipe completion.
-    - `.EUt()`: Takes a numerical value represesnting an EU amount. Positive values will make the recipe consume energy per tick, negative ones will make it generate energy per tick.
+    - `.inputEU()`:  
+      Makes the recipe consume a lump sum of EU to start the recipe. Most often seen in fusion reactor recipes.
+    - `.outputEU()`:  
+      Makes the recipe produce a lump sum of EU upon recipe completion.
+  - `.EUt()`:  
+    Takes a numerical value representing an EU amount. Positive values will make the recipe consume energy per tick,
+    negative ones will make it generate energy per tick.
 - More granular functionality:
-    - `.perTick()`: Using this will enable you to control whether a recipe input/output is consumed/produced per tick the recipe is running or all at once at recipe start/end. Set to true with `.perTick(true)` to make the recipe builder consider any following input/output calls as per-tick. Remember to set the value to false with `.perTick(false)` after the calls you intend to be per-tick, to prevent behaviour you don't want!
+    - `.perTick()`:  
+      Using this will enable you to control whether a recipe input/output is consumed/produced per tick the recipe is
+      running or all at once at recipe start/end. Set to true with `.perTick(true)` to make the recipe builder consider
+      any following input/output calls as per-tick. Remember to set the value to false with `.perTick(false)` after the
+      calls you intend to be per-tick, to prevent behaviour you don't want!
+
 
 ### The Research System
-GTCEu has Research System which allows for adding extra requirements to recipes such as: Scanner Research, Station Research and Computation.
+
+GTCEu has Research System which allows for adding extra requirements to recipes such as:  
```

Comment:
Since its modern docs, probably call it GTCEu Modern instead of just GTCEu here? I noticed its called that in earlier instances too